zipline-gradle-plugin/src/test/kotlin/app/cash/zipline/gradle/ZiplineCompilerTest.kt

```diff
@@ -85,6 +86,53 @@ class ZiplineCompilerTest {
     }
   }
 
+  @Test
+  fun `incremental compile`() {
+    val rootProject = "src/test/resources/incremental"
+    val outputDir = File("$rootProject/base/build/zipline")
+
+    // Clean up dir from previous runs
+    if (outputDir.exists()) {
+      outputDir.deleteRecursively()
+    }
```

Comment:
Should probably use a JUnit temporary directory rule here to ensure a clear slate

Reply:
Maybe I'll do this as a separate PR, since really we should do this for all test, not just the incremental ones.